src/toil/provisioners/abstractProvisioner.py

```diff
@@ -813,13 +817,11 @@ def add_toil_service(self, config: InstanceConfiguration, role: str, keyPath: st
                 -v /etc/kubernetes:/etc/kubernetes \\
                 -v /etc/kubernetes/admin.conf:/root/.kube/config \\
                 # Pass in a path to use for singularity image caching into the container
+                {"-e TOIL_KUBERNETES_PRIVILEGED=True --privileged" if self.enable_fuse else 
+                "--security-opt seccomp=unconfined --security-opt systempaths=unconfined"} \\
                 -e TOIL_KUBERNETES_HOST_PATH=/var/lib/toil \\
                 -e SINGULARITY_CACHEDIR=/var/lib/toil/singularity \\
```

Comment:
The comment for `SINGULARITY_CACHEDIR` has gotten separated from `SINGULARITY_CACHEDIR` here and should be reattached, instead of adding more lines between them.